INSTRUCTIONS2RUN.md

```diff
@@ -0,0 +1,31 @@
+[toc]
+# gms-artic in ngp-gms
```

Comment:
Preferably integrated into the README.md

Reply:
okay, I will do it.

Reply:
Done!